packages/babel-core/src/config/files/configuration.js

```diff
@@ -299,6 +299,20 @@ const readIgnoreConfig = makeStaticFileCache((filepath, content) => {
   };
 });
 
+export function* resolveShowConfigPath(
+  dirname: string,
+): Handler<string | null> {
+  const targetPath = process.env.BABEL_SHOW_CONFIG_FOR_PATH;
```

Comment:
Do you think the following candidates are easier for typing?
```
BABEL_SHOW_CONFIG_FOR
BABEL_CONFIG_FOR
```

Reply:
I slightly prefer `BABEL_SHOW_CONFIG_FOR`